IbisPlugins/GPUVolumeReconstructionAPITest/gpuvolumereconstructionapitestplugininterface.cpp

```diff
@@ -62,15 +62,18 @@ QWidget * GPUVolumeReconstructionAPITestPluginInterface::CreateFloatingWidget()
 
              //Construct ITK Matrix corresponding to VTK Local Matrix
             reconstructor->SetTransform( acq->GetLocalTransform()->GetMatrix() );
-            reconstructor->ReconstructVolume();
+            reconstructor->start();
+            reconstructor->wait();
```

Comment:
At the moment, it is useless to perform the reconstruction in a separate thread as you are asking the main thread to wait for the reconstruction thread to finish, so the main (GUI) thread is blocked while the reconstruction is running.